### PR TITLE
Update i18n.yaml

### PR DESCRIPTION
**Pull request by yukulele**
improve french

### CODE REVIEW

File: data/i18n.yaml [hunk 1]
```diff
@@ -14,9 +14,9 @@ fr_fr:
   where: Où X=
   getCode: "Récupérer le code :"
   share: Partager cette page
-  selectTheme: "Sélectionne un thème:"
-  lightTheme: lumière
-  darkTheme: foncé
+  selectTheme: "Sélectionner un thème :"
+  lightTheme: clair
```
Comment (Menelion) [inline data/i18n.yaml:18]: 👍🏻 👍🏻

**Review by Menelion — APPROVED** [review]
Merci bien!

**Comment by Menelion** [comment]
@adambard Oh, I have no permissions here :) LGTM.

**Comment by adambard** [comment]
Merci beaucoup!

File: data/i18n.yaml [hunk 2]
```diff
@@ -14,9 +14,9 @@ fr_fr:
...
+  darkTheme: sombre
```
Comment (Soheilafsha) [inline data/i18n.yaml:19]: Active

**Pull request merged**